src/skydb_internal.ts

```diff
@@ -0,0 +1,192 @@
+import {
+  deriveEncryptedFileKeyEntropy,
+  deriveEncryptedFileTweak,
+  decryptJSONFile,
+  ENCRYPTED_JSON_RESPONSE_VERSION,
+  EncryptedJSONResponse,
+  getOrCreateSkyDBRegistryEntry,
+  JsonData,
+  SkynetClient,
+  encryptJSONFile,
+  MAX_REVISION,
+  RegistryEntry,
+  signEntry,
+} from "skynet-js";
+
+import { deriveRootPathSeed, genKeyPairFromSeed } from "./crypto";
+import { deriveEncryptedPathSeedForRoot } from "./encrypted_files";
+import { log, validateObject, validateString } from "./util";
+
+/**
+ * Gets Encrypted JSON at the given path through MySky.
+ *
+ * @param client - The Skynet client.
+ * @param seed - The user seed.
```

Comment:
That's a good question, indeed. It will be the MySky root seed and not an account-specific seed. I will update the relevant docstrings to make this distinction more clear for future readers of the code.